src/utils.c

```diff
@@ -99,14 +99,17 @@ SEXP allNAR(SEXP x) {
  *   existing columns for character
  *   optionally check for no duplicates
```

Comment:
```suggestion
 *   optionally (check_dups) check for no duplicates
 *   optionally (skip_absent) skip (return 0) for numbers outside the range or not naming extant columns
```

Reply:
in other places, `skip_absent` will _drop_ the missing columns, but I'm not sure this routine should do that -- it's nice to have `length(colnamesInt(x, cols)) == length(cols)` respected. Then the caller can figure out what to do with the 0 entries, e.g. drop them.

(no action item here, just thinking aloud about why I think returning 0 is OK)